test/unit/credit_card_methods_test.rb

```diff
@@ -192,6 +192,14 @@ def test_should_detect_mada_card
     assert_equal 'mada', CreditCard.brand?('9682040000000000')
   end
 
+  def test_should_detect_bp_plus_card
+    assert_equal 'bp_plus', CreditCard.brand?('7051 211111111 111')
+    assert_equal 'bp_plus', CreditCard.brand?('7050 111111111 111')
```

Comment:
Based on the documentation I'm reading, I think these two test cases are not correct. The documentation shows that the format for cards starting with `705N` is `705N NNNNNNNN NNNNN`, with the final digit being a luhn check number. I am going to ask for more information about this, but at the very least I think the format after dropping the final number would be `7050 11111111 1111` and not `7050 111111111 111`.